services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -1261,7 +1267,8 @@ def compute_config_parquet_and_info_response(
             parquet_operations = copy_parquet_files(builder)
             logging.info(f"{len(parquet_operations)} parquet files to copy for {dataset=} {config=}.")
             validate = ParquetFileValidator(max_row_group_byte_size=max_row_group_byte_size_for_copy).validate
-            fill_builder_info(builder, hf_endpoint=hf_endpoint, hf_token=hf_token, validate=validate)
+            with patch("huggingface_hub.hf_file_system.http_backoff", http_backoff_with_timeout):
```

Comment:
Feels quite hacky :confused: 
No solution to do that differently? What's the default timeout and what the value you would prefer?

Reply:
Default is None (can hang indefinitely). In `datasets` we use 100 by default.

I'm using this hack right now to validate this issue, we can see later for a proper fix on the hfh side